src/functionalTest/resources/application.properties

```diff
@@ -33,6 +33,7 @@ case.orchestration.service.base.uri=#{environment["TEST_URL"].replaceFirst("http
 case.orchestration.petition-issued.context-path=/petition-issued
 case.orchestration.maintenance.submit.context-path=/submit
 case.orchestration.maintenance.case.context-path=/case
+case.orchestration.maintenance.get-case.context-path=/case
 case.orchestration.maintenance.update.context-path=/updateCase
```

Comment:
Do you not just want to use the `case.orchestration.maintenance.case.context-path` above since they are the same?

Reply:
done